compiler/mx.compiler/suite.py

```diff
@@ -1,5 +1,5 @@
 suite = {
-  "mxversion" : "5.309.2",
+  "mxversion" : "5.316.0",
```

Comment:
@dougxc I can't access 5.316.0, mx `master` seems to be on 5.315.2 https://github.com/graalvm/mx/blob/dd0a0c7a81c412722650f99e9123707a99e1f088/mx.py#L17796

Reply:
It's actually going to be easier if you test your changes in a separate branch. I can then later graft that onto my branch (or we can merge it independently).